### PR TITLE
Further improvements and clean ups

### PR DESCRIPTION
**Pull request by mbeckerle**
Change type names to reflect units of measure (a useful practice makes things a bit more self-documenting)

Added tests to subsume all the ones in the Owl DFDL training.

Some renaming to make things more uniform. Tests of single messages - the files and tests have "msg" in their names. Tests of files of multiple messages have "file" in their names.

### CODE REVIEW

**Comment by mbeckerle** [comment]
@jadams-tresys Any comments at all on fakeTDL are welcome. 

From here on I will make sure any changes to it are PRs so you are kept aware.

The training will essentially build this schema bit by bit. Minor differences.

File: src/fakeTDL.dfdl.xsd [hunk 1]
```diff
@@ -28,11 +28,11 @@
 
   <xs:annotation>
     <xs:appinfo source="http://www.ogf.org/dfdl/">
-      <dfdl:format ref="fakeTDL:fakeTDL-base"/>
+      <dfdl:format ref="fakeTDL:baseFormat"/>
     </xs:appinfo>
   </xs:annotation>
 
-  <xs:element name="fakeTDL" type="fakeTDL:fakeTDLType" dfdl:ref="fakeTDL:fakeTDL-fixedLengthMessage">
+  <xs:element name="fakeTDL" type="fakeTDL:fakeTDLType" dfdl:ref="fakeTDL:msg">
```
Comment (olabusayoT) [inline src/fakeTDL.dfdl.xsd:35]: We are still reffing this element down below in fakeTDLFile? See #7

**Review by olabusayoT — CHANGES_REQUESTED** [review]
#7 is not yet taken care of

File: src/fakeTDL.dfdl.xsd (hunk 1 above)
Comment (mbeckerle) [reply src/fakeTDL.dfdl.xsd:35]: Ah. I fixed it in the training materials, not here. 
Thanks.

Comment (mbeckerle) [reply src/fakeTDL.dfdl.xsd:35]: Pushed fix for this.

**Review by olabusayoT — APPROVED** [review]
+1

**Review by jadams-tresys — APPROVED** [review]
+1

**Pull request merged**